dot/services.go

```diff
@@ -36,17 +36,18 @@ import (
 	"github.com/ChainSafe/gossamer/lib/utils"
 )
 
-// createRPCServiceParams struct to implements the Parameter Object Pattern: https://refactoring.guru/introduce-parameter-object
-type createRPCServiceParams struct {
-	cfg         *Config
-	ns          *runtime.NodeStorage
-	stateSrvc   *state.Service
-	coreSrvc    *core.Service
-	networkSrvc *network.Service
-	bp          modules.BlockProducerAPI
-	sysSrvc     *system.Service
-	finSrvc     *grandpa.Service
-	syncer      *sync.Service
+// rpcServiceSettings struct to implements the Parameter Object Pattern:
+// https://refactoring.guru/introduce-parameter-object
```

Comment:
nit I think we can remove this, the type name is quite self explanatory :thinking: 

```suggestion
```

Reply:
Done in c2e3d69